src/app/tests/suites/certification/Test_TC_SC_5_2.yaml

```diff
@@ -102,39 +106,77 @@ tests:
               - name: GroupID
                 value: 0x0103
               - name: GroupName
-                value: "Test Group"
+                value: "Test Group 0103"
+      response:
+          values:
+              - name: Status
+                value: 0
+              - name: GroupID
+                value: 0x0103
 
-    - label: "TH sends ViewGroup command using group messaging"
+    - label:
+          "TH sends AddGroup command again but using group messaging (0x0103)"
       PICS: G.S.F00
-      command: "ViewGroup"
       groupId: 0x0103
+      command: "AddGroup"
       arguments:
           values:
               - name: GroupID
-                value: 0x0103
+                value: 0x0101
+              - name: GroupName
+                value: "Test Group 0101"
+
+    - label:
+          "TH sends AddGroup command again but using group messaging (0x0103)"
+      PICS: "!(G.S.F00)"
```

Comment:
Why do we need this separate bit conditioned on the PICS check?  If the server does not support G.S.F00, then it will just ignore the GroupName field, so we can send it as "Test Group 0101" in both cases.... and then we don't need the two cases at all.